.ci/on-schedule.sh

```diff
@@ -225,5 +284,5 @@ if [ "$COMMIT" = true ]; then
     for branch in "${DELETE_BRANCHES[@]}"; do
         git_push_args+=(":$branch")
     done
-    git push --atomic origin HEAD:main +refs/tags/scheduled "${git_push_args[@]}"
+    git push --atomic "$REPO_URL" HEAD:main +refs/tags/scheduled "${git_push_args[@]}"
```

Comment:
Why does this need to be changed? This breaks github actions.

```suggestion
    git push --atomic origin HEAD:main +refs/tags/scheduled "${git_push_args[@]}"
```